extension/chrome/settings/modules/backup-status-module.ts

```diff
@@ -81,10 +80,42 @@ export class BackupStatusModule extends ViewModule<BackupView> {
   }
 
   private actionShowManualBackupHandler = async () => {
+    const primaryKeys = await KeyStore.getAllWithOptionalPassPhrase(this.view.acctEmail);
+    if (primaryKeys.length > 1) {
+      this.renderPrvKeysBackupSelection(primaryKeys);
+    }
     this.view.displayBlock('module_manual');
     $('h1').text('Back up your private key');
   }
 
+  private renderPrvKeysBackupSelection = (primaryKeys: ExtendedKeyInfo[]) => {
+    for (const primaryKi of primaryKeys) {
+      const dom = `
+      <div class="mb-20">
+        <div class="details">
+          <label>
+            <input class="input_prvkey_backup_checkbox" type="checkbox" />
+            <div class="display_inline_block">
+              <p class="m-0">Email: <span class="prv_email">${primaryKi.emails}</span> with fingerprint <span class="prv_fingerprint green">${primaryKi.fingerprints}</span></p>
+            </div>
+          </label>
+        </div>
+      </div>
+      `.trim();
+      $('.key_backup_selection').append(dom);
+    }
+    $('.input_prvkey_backup_checkbox').click((event) => {
+      const email = String($($($(event.target).siblings()[0]).children()[0]).children()[0].innerText);
```

Comment:
Instead of this [spaghetti code](https://en.wikipedia.org/wiki/Spaghetti_code) I'd recommend this:

```html
<input class="input_prvkey_backup_checkbox" type="checkbox" data-email="${primaryKi.emails}"/>
```

```js
const email = $(event.target).data('email');
```